README.md

```diff
@@ -1,30 +1,39 @@
 # What is this repository for?
 
 To store the docker BUILD scripts of ONNX related docker images.
-- onnx-docker-cpu
-- onnx-docker-gpu
-- [onnx-ecosystem](onnx-ecosystem): Jupyter notebook environment for getting started quickly with ONNX models, ONNX converters, and inference using ONNX runtime.
+- [onnx-docker-cpu](onnx-docker-cpu/Dockerfile): Image with ONNX, PyTorch, Tensorflow support
+- [onnx-docker-gpu](onnx-docker-gpu/Dockerfile): Image with ONNX, PyTorch (CUDA), Caffe2 support
+- [onnx-ecosystem](onnx-ecosystem): Jupyter notebook environment for getting started quickly with ONNX models, ONNX converters, and inference using ONNX Runtime.
```

Comment:
is onnx-ecosystem a superset of the onnx-docker-cpu?

Reply:
Functionally yes, with a few caveats:
- onnx-docker-cpu is used for the ONNX backed scoreboard: http://onnx.ai/backend-scoreboard/
- It uses onnx-tensorflow instead of tf2onnx
- It builds pytorch / onnx from source instead of from released packages